webpack/assets/javascripts/react_app/components/charts/spec/fixtures.js

```diff
@@ -0,0 +1,14 @@
+export const statisticsData = [
+  {
+    id: 'operatingsystem',
+    title: 'OS Distribution',
+    url: 'statistics/operatingsystem',
```

Comment:
Yes, and It was a dilema, i thought that the way components are currently built, data-specific tests may belong in the chartbox component, since the list has a job of rendering x panels for x items in the state.
tests of this component can be improved for sure, including a mount test to see if actions are fired properly and empty store gets proper values when given data.